---
title: >-
  Data from Construction of a High-Density American Cranberry (Vaccinium
  macrocarpon Ait.) Composite Map Using Genotyping-by-Sequencing for
  Multi-pedigree Linkage Mapping
created: '2020-11-10T16:26:31.450843'
modified: '2020-11-10T16:26:31.450856'
state: active
type: dataset
tags:
  - Cranberry Full Sib Populations
groups: []
csv_url: >-
  http://www.g3journal.org/highwire/filestream/483420/field_highwire_adjunct_files/1/FileS2.csv
json_url: ''
layout: post

---
<p>The American cranberry (<em>Vaccinium macrocarpon</em> Ait.) is a recently domesticated, economically important, fruit crop with limited molecular resources. New genetic resources could accelerate genetic gain in cranberry through characterization of its genomic structure and by enabling molecular-assisted breeding strategies. To increase the availability of cranberry genomic resources, genotyping-by-sequencing (GBS) was used to discover and genotype thousands of single nucleotide polymorphisms (SNPs) within three interrelated cranberry full-sib populations. Additional simple sequence repeat (SSR) loci were added to the SNP datasets and used to construct bin maps for the parents of the populations, which were then merged to create the first high-density cranberry composite map containing 6073 markers (5437 SNPs and 636 SSRs) on 12 linkage groups (LGs) spanning 1124 cM. Interestingly, higher rates of recombination were observed in maternal than paternal gametes. The large number of markers in common (mean of 57.3) and the high degree of observed collinearity (mean Pair-wise Spearman rank correlations &gt;0.99) between the LGs of the parental maps demonstrates the utility of GBS in cranberry for identifying polymorphic SNP loci that are transferable between pedigrees and populations in future trait-association studies. Furthermore, the high-density of markers anchored within the component maps allowed identification of segregation distortion regions, placement of centromeres on each of the 12 LGs, and anchoring of genomic scaffolds. Collectively, the results represent an important contribution to the current understanding of cranberry genomic structure and to the availability of molecular tools for future genetic research and breeding efforts in cranberry.</p>

