---
title: >-
  Data from Variation in genome size and karyotype among closely-related aphid
  parasitoids (Hymenoptera Aphelinidae)
created: '2020-11-10T16:26:43.570700'
modified: '2020-11-10T16:26:43.570711'
state: active
type: dataset
tags:
  - Aphelinidae
  - Aphelinus
  - Flow Cytometry
  - Genome Size
  - Karotype
  - Np304
  - Parasitoid
groups: []
csv_url: >-
  https://data.nal.usda.gov/system/files/Aphelinus_karyotypes_relative_length_centromeric_index.csv
json_url: ''
layout: post

---
<p>We measured genome sizes and determined the karyotypes of nine species of aphid parasitoids in the genus Aphelinus. We found large differences in genome size and karyotype between Aphelinus species, which is surprising given the similarity in their morphology and life history. Genome sizes estimated from flow cytometry were larger for species in the mali complex than those for the species in the daucicola and varipes complexes. Haploid karyotypes of the daucicola and mali complexes comprised five metacentric chromosomes of similar size, whereas those of the varipes complex had four chromosomes, including a larger and a smaller metacentric chromosome and two small acrocentric chromosomes or a large metacentric and three smaller acrocentric chromosomes. Total lengths of female haploid chromosome sets correlated with genome sizes estimated from flow cytometry. Phylogenetic analysis of karyotypic variation revealed a chromosomal fusion together with pericentric inversions in the common ancestor of the varipes complex and further pericentric inversions in the clade comprising Aphelinus kurdjumovi and Aphelinus hordei. Fluorescence in situ hybridization with a 28S ribosomal DNA probe revealed a single site on chromosomes of the haploid karyotype of Aphelinus coreae. The differences in genome size and total chromosome length between species complexes matched the phylogenetic divergence between them.</p>
<p><strong>Materials and Methods</strong></p>
<p><strong>Specimens</strong><br />
The parasitoid species studied and the sources of the colonies are listed in the data file "<a href="https://data.nal.usda.gov/dataset/data-variation-genome-size-and-karyotype-among-closely-related-aphid-parasitoids-0">Aphelinus_species_studied.csv</a>". These colonies were reared on aphids at the USDA-ARS, Beneficial Insect Introductions Research Unit, in Newark, Delaware, USA. Females of the yellow-white strain of Drosophila melanogaster (Meigen, 1830) (stock number 1495, obtained from the Bloomington Drosophila Stock Center at Indiana University, <a href="http://flystocks.bio.indiana.edu">http://flystocks.bio.indiana.edu</a>) were used as internal controls for flow cytometry. All institutional and national guidelines for the care and use of laboratory animals were followed.</p>
<p><strong>Flow cytometry</strong><br />
Live Aphelinus were sexed, flash frozen in liquid nitrogen, and stored at −80°C. To estimate genome sizes, we used the flow cytometry protocol described by Hanrahan and Johnston (2011) and Hare and Johnston (2011). We dissected heads from both males and females of the Aphelinus species in cold Galbraith buffer (Galbraith et al. 1983). Heads of female D. melanogaster were used as internal standards (1C = 175 Mb or 0.17 pg). To release the nuclei from cells, 15 female Aphelinus heads and one female Drosophila head, were ground together in one milliliter of cold Galbraith buffer using 15 strokes of the "A" pestle in a 2-ml Kontes Dounce tissue grinder. Three to six replicates were done for females and males of each species, but because male genome sizes were too close to that of Drosophila, we used heads from females of the same parasitoid species as internal standards for males. The samples were passed through a 35 micron filter and then stained with 40 parts per million of propidium iodide in the dark for 3-5 hours at 4°C. Samples were analyzed on a Becton Dickinson FACSCalibur Flow Cytometer with laser excitation at 488 nm. Red fluorescence from the propidium iodide was detected using an FL2 filter. The haploid content of DNA in megabases (Mb) was calculated for each Aphelinus sample from the ratio of mean fluorescence of the sample to mean fluorescence of the standard times the genome size of the standard. We report genome size estimates in megabases, but also give estimates in picograms (pg) calculated by dividing the amount of DNA in Mb by the standard 1C value of 978 Mb.</p>
<p><strong>Karyotypes</strong><br />
Chromosome preparations were made from cerebral ganglia of prepupae using a modified version of the technique in Imai et al. (1988). Wasps were dissected in 0.5% hypotonic sodium citrate solution containing 0.005% colchicine, and the tissues were incubated in fresh solution for ~30 minutes at room temperature. The material was transferred to a pre-cleaned microscope slide using a Pasteur pipette and gently flushed with Fixative I (glacial acetic acid: absolute ethanol: distilled water 3:3:4). Tissues were disrupted in an additional drop of Fixative I using dissecting needles. Another drop of Fixative II (glacial acetic acid: absolute ethanol 1:1) was then applied to the center of the area and blotted off the edges of the slide. The slide was air dried for ~30 minutes at room temperature. For conventional staining, preparations were stained with freshly prepared 3% Giemsa solution in 0.05M Sørensen's phosphate buffer (Na2HPO4 + KH2PO4, pH 6.8). Mitotic divisions were studied and photographed using an optic microscope Zeiss Axioskop 40 FL fitted with a digital camera AxioCam MRc (Carl Zeiss, Oberkochen, Germany). To obtain karyograms, the resulting images were processed with image analysis programs: Zeiss AxioVision version 3.1 and Adobe Photoshop version 8.0. Mitotic chromosomes were measured for 5-19 cells in 1-4 wasps per species using Adobe Photoshop. We report total length of all chromosomes in each karyotype; for diploid sets, we divided total length by two to make the values comparable to haploid sets. We also report relative lengths (RL; 100 x length of each chromosome divided by total length of the set) and centromeric indices (CI; 100 x length of shorter arm divided by total length of a chromosome). Chromosomes were classified into metacentric (M) or acrocentric (A) according to the guidelines in Levan et al. (1964).</p>

